src/products/searchplugins/setup_searchplugins.md

```diff
@@ -1,193 +1,119 @@
 
-You can’t copy the full URL from the address bar since it will encode UTF-8 characters, e.g. `https://ur.wikipedia.org/wiki/%D8%AE%D8%A7%D8%B5:%D8%AA%D9%84%D8%A7%D8%B4`, so it’s faster to just copy only part of the URL and fixing the rest by hand.
+You can’t copy the full URL from the address bar since it will encode UTF-8 characters, e.g. `https://ur.wikipedia.org/wiki/%D8%AE%D8%A7%D8%B5:%D8%AA%D9%84%D8%A7%D8%B4`, so it’s faster to just copy only part of the URL and fix the rest by hand.
 
 **Description:** You will need localizers to provide the translation for `Wikipedia, the Free Encyclopedia`.
 
-**shortName:** In the .xml file there’s an attribute called `shortName`. For English is:
+**Name:** In the localization file there’s an attribute called `extensionName`. For English is:
 
-```XML
-<ShortName>Wikipedia</ShortName>
+```JSON
+"extensionName": {
+  "message": "Wikipedia (en)"
+},
 ```
 
-When you visit a page that exposes a searchplugin, like Wikipedia, Firefox checks if you already have one installed with the same `shortName`: if you don’t, the UI will suggest you to install it by displaying a small green + sign on the magnifying glass icon in the search bar. To avoid this, the searchplugin shipping in Firefox needs to match the shortName of the searchplugin available from Wikipedia. Visit https://gn.wikipedia.org and press `CTRL+U` to view the source code of the page. At the beginning you will find a line looking like this:
+To get the localized name, visit https://gn.wikipedia.org and press `CTRL+U` to view the source code of the page. At the beginning you will find a line looking like this:
 
 ```HTML
 <link rel="search" type="application/opensearchdescription+xml" href="/w/opensearch_desc.php" title="ویکیپیڈیا (ur)"/>
 ```
 
-The shortName to use in your .xml file is `ویکیپیڈیا (ur)`. Copying and pasting RTL languages is particularly tricky.
-
-## Creating a patch for review
-
-Once files are ready, follow the instructions available in [this document](../../tools/mercurial/creating_mercurial_patch.md) to create a patch, submit it for review, and land it.
+The *Name* to use in your localization is `ویکیپیڈیا (ur)`. Copying and pasting RTL languages is particularly tricky.
 
 ## Setting up files for locale’s repository
 
 ### region.properties
 
-region.properties is stored in `/browser/chrome/browser-region` for Firefox desktop (`/mobile/chrome` for Firefox for Android), and it contains information about protocol handlers. You can use these files as a base:
-* For Firefox desktop [this region.properties model](files/desktop_region.properties).
-* For Firefox for Android [this region.properties model](files/mobile_region.properties).
+region.properties is stored in `/browser/chrome/browser-region` for Firefox desktop, and it contains information about protocol handlers. You can use [this region.properties model](files/desktop_region.properties) as a base .
```

Comment:
Extra space before the period at the end.

Reply:
Thanks, fixed 👍